lessons/basics/modules.md

```diff
@@ -194,9 +202,28 @@ import List, only: :functions
 import List, only: :macros
 ```
 
+It is also worth noting that import is lexical in scope.
```

Comment:
Misunderstood original comments concerns. I'm for backticks around them, I think, but dont believe we need arity (for things like `use` `require` `import` `__using__` etc.)